packages/compass-status/lib/index.js

```diff
@@ -0,0 +1,6 @@
a.a.string,animation:a.a.bool,sidebar:a.a.bool,subview:a.a.any,subviewStore:a.a.object,subviewActions:a.a.object});var y=v,_=n(3),x=n.n(_),w=n(10),A=n.n(w);var j=x.a.createActions(["showProgressBar","showIndeterminateProgressBar","enableProgressTrickle","disableProgressTrickle","setMessage","clearMessage","setSubview","setSubviewStore","setSubviewActions","clearSubview","enableModal","disableModal","configure","hide","done","setGlobalAppRegistry"]);var T,S,k,D=x.a.createStore({mixins:[A.a.store],listenables:j,onActivated(e){this.setGlobalAppRegistry(e)},init(){this._trickleTimer=null},getInitialState:()=>({visible:!1,progressbar:!1,progress:0,modal:!1,animation:!1,message:"",subview:null,subviewStore:null,subviewActions:null,trickle:!1,globalAppRegistry:null}),setGlobalAppRegistry(e){this.setState({globalAppRegistry:e})},showProgressBar(){this.showIndeterminateProgressBar()},showIndeterminateProgressBar(){this.disableProgressTrickle(),this.setState({visible:!0,progressbar:!0,progress:100,trickle:!1})},configure(e){e.trickle?this.enableProgressTrickle():this.disableProgressTrickle(),this.setState(e)},enableProgressTrickle(){this._trickleTimer||(this._trickleTimer=setInterval(()=>{const e=Math.min(98,this.state.progress+1);this.setState.call(this,{progress:e})},600),this.setState({trickle:!0}))},disableProgressTrickle(){null!==this._trickleTimer&&(clearInterval(this._trickleTimer),this._trickleTimer=null),this.setState({trickle:!1})},setMessage(e){this.setState({visible:!0,message:e})},clearMessage(){this.setState({message:""})},showAnimation(){this.setState({visible:!0,animation:!0})},hideAnimation(){this.setState({animation:!1})},setSubview(e){this.setState({subview:e})},setSubviewStore(e){this.setState({subviewStore:e})},setSubviewActions(e){this.setState({subviewActions:e})},clearSubview(){this.setState({subview:null,subviewStore:null,subviewActions:null})},enableModal(){this.setState({modal:!0})},disableModal(){this.setState({modal:!1})},hide(){this.disableProgressTrickle(),this.setState(this.getInitialState())},done(){this.disableProgressTrickle(),this.setState({progress:100,animation:!1,message:"",subview:null,subviewStore:null,subviewActions:null}),this.hide()}});function V(){return(V=Object.assign||function(e){for(var t=1;t<arguments.length;t++){var n=arguments[t];for(var r in n)Object.prototype.hasOwnProperty.call(n,r)&&(e[r]=n[r])}return e}).apply(this,arguments)}class O extends r.Component{render(){return i.a.createElement(s.StoreConnector,{store:D},i.a.createElement(y,V({actions:j},this.props)))}}k="StatusPlugin",(S="displayName")in(T=O)?Object.defineProperty(T,S,{value:k,enumerable:!0,configurable:!0,writable:!0}):T[S]=k;var P=O,E=n(5);const N={name:"Status",component:P};function M(e){e.registerRole("Application.Status",N),e.registerAction("Status.Actions",j),e.registerStore("Status.Store",D)}function C(e){e.deregisterRole("Application.Status",N),e.deregisterAction("Status.Actions"),e.deregisterStore("Status.Store")}t.default=P}])}));
```

Comment:
These changes are entirely unrelated, right?

Reply:
Yeah, I don't know where they are coming from, will remove it shortly